app/_includes/plugins/schemas/field.html

```diff
@@ -96,6 +108,22 @@ <h4 id="{{ include.field.anchor }}">
   <div class="field-description-and-children">
     <div class="field-description">{{ include.field.description | markdownify }}</div>
 
+    {% if include.field.deprecated? %}
+      <div class="field-deprecation">
+        <blockquote class="important">
+          <strong>Deprecation notice:</strong>
+          {% if include.field.deprecation['message'] -%}
+            <span class="field-deprecation__message">{{ include.field.deprecation['message'] }}.</span>
+          {%- endif -%}
+          {%- if include.field.deprecation['removal_in_version'] -%}
+            <span class="field-deprecation__version">
+              This field will be removed in version: {{ include.field.deprecation['removal_in_version'] }}.
```

Comment:
```suggestion
              This field is planned to be removed in version {{ include.field.deprecation['removal_in_version'] }}.
```
rephrase to "planned" because we can't guarantee and don't want to commit to this number. It often changes.